dora/core/server/worker/src/main/java/alluxio/worker/http/HttpServerHandler.java

```diff
@@ -156,6 +156,7 @@ private HttpResponseContext doListFiles(HttpRequest httpRequest, HttpRequestUri
       for (URIStatus uriStatus : uriStatuses) {
         String type = uriStatus.isFolder() ? "directory" : "file";
         ResponseFileInfo responseFileInfo = new ResponseFileInfo(type, uriStatus.getName(),
+            uriStatus.getPath(), uriStatus.getUfsPath(), uriStatus.getLastModificationTimeMs(),
```

Comment:
@JiamingMai @dbw9580 does this work when multi-mounts is enabled?

Reply:
> @JiamingMai @dbw9580 does this work when multi-mounts is enabled?

If multi-mounts is used, then user needs to specify the full path when requesting this RESTful API. For example, `curl -X GET http://localhost:28080/v1/files?path=s3a://test-bucket/`.

Reply:
this is worker side code, all Alluxio-namespace URLs should already be resolved to UFS URLs, so it should also work with multi-mounts.

Reply:
So, the result list of files will have different paths or UFS paths with that in FUSE.
Applications/Users have to manually do another translation from UFS path to the path in FUSE.
I am not sure if this is the expected behavior.

Reply:
> this is worker side code, all Alluxio-namespace URLs should already be resolved to UFS URLs, so it should also work with multi-mounts.

Yes, I know that.

So, the user will have a tree of files on client side FUSE mount point.
But the user has to use a UFS full path to do such listing on this API. And the result {path, UFS path} does not automatically map to any file on client.
The purpose to do such a listing is to have a full list of some directory and then filter out some files to be used in next step.
If the result from this API can not be mapped directly to the client view of namespace, the user can not use such list to do further data processing.

Reply:
then I think the user should access the restful api from a proxy server, which has the mount table and can convert UFS URLs back to Alluxio URLs.